.github/workflows/security.yml

```diff
@@ -31,13 +31,36 @@ jobs:
         with:
           ref: ${{ matrix.version }}
 
-      - name: Run Trivy vulnerability scanner in repo mode
-        uses: aquasecurity/trivy-action@master
+      - name: Install Trivy
+        uses: canonical/lxd/.github/actions/install-trivy@main
+
+      - name: Download Trivy DB
+        id: db_download
+        run: trivy fs --download-db-only --cache-dir /home/runner/vuln-cache
+        continue-on-error: true
+
+      - name: Use previous downloaded database
+        if: ${{ steps.db_download.outcome == 'failure' }}
+        uses: actions/cache/restore@3624ceb22c1c5a301c8db4169662070a689d9ea8 # v4.1.1
```

Comment:
Where does this point to? Which repo?

Reply:
Do you mean `actions/cache`?

Reply:
Ah my bad, I was referring to the commit `3624...` and was thinking you restore the cache of a previous action ran for this commit. Never mind :)

Reply:
Actually it's the version of the action.

Reply:
And you restore whatever is stored under `trivy-latest-cache`.